### PR TITLE
[zh-cn]: update the translation of Navigator.getGamepads() method

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/getgamepads/index.md [hunk 1]
```diff
@@ -1,29 +1,42 @@
...
+  sourceCommit: acfe8c9f1f4145f77653a2bc64a9744b001358dc
 ---
 
-{{APIRef("Gamepad API")}}{{SeeCompatTable}}
+{{APIRef("Gamepad API")}}{{securecontext_header}}
 
-调用 **`Navigator.getGamepads()`** 方法会返回一个数组：第一个值为 `null` ，其他的值均为 {{ domxref("Gamepad") }} 对象，表示每一个与设备连接的游戏手柄。所以如果没有连接任何游戏手柄，这个方法将只会返回 `null`。
+**`Navigator.getGamepads()`** 方法返回一个包含 {{domxref("Gamepad")}} 数组对象，每个对象代表与设备相连的一个游戏手柄。
+
+如果游戏手柄在过程断开连接，否则数组中的元素可能为 `null`，但剩余的游戏手柄的索引保持不变。
```
Comment (T34-active) [inline files/zh-cn/web/api/navigator/getgamepads/index.md:12]:
```suggestion
如果游戏手柄在过程断开连接，则数组中的元素可能为 `null`，但剩余的游戏手柄的索引保持不变。
```

File: files/zh-cn/web/api/navigator/getgamepads/index.md [hunk 2]
```diff
@@ -1,29 +1,42 @@
...
 ```
 
-## 样例
+### 参数
+
+无。
+
+### 返回值
+
+一个由 {{domxref("Gamepad")}} 对象的 {{jsxref("Array")}}，最终可能为空。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/getgamepads/index.md:26]:
```suggestion
一个 {{domxref("Gamepad")}} 对象的{{jsxref("Array", "数组", "", 1)}}，最终可能为空。
```

File: files/zh-cn/web/api/navigator/getgamepads/index.md [hunk 3]
```diff
@@ -1,29 +1,42 @@
...
+
+### 异常
+
+- `SecurityError` {{domxref("DOMException")}}
+  - : 使用此功能被[权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)阻止。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/getgamepads/index.md:31]:
```suggestion
  - : [权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)阻止了此特性的使用。
```

**Review by yin1999 — COMMENTED** [review]
nit:

**Comment by T34-active** [comment]
@yin1999 Thank you!

**Review by yin1999 — APPROVED** [review]

**Pull request merged**